docs/_diagrams/arch-dark.uml

```diff
@@ -0,0 +1,3 @@
+@startuml
+!include arch.uml
+@enduml
```

Comment:
Q: Is the `arch-dark.uml` really necessary? Can't we do something like:

```
${plantuml}/bin/plantuml -tsvg -darkmode arch.uml -o ../_static/arch-dark.svg
```

Reply:
Ah, no, it won't let me. Doing that creates the SVG in this path:

```
/_static/arch-dark.svg/arch.svg
```

I tried using `-filename` ([see the docs](https://plantuml.com/en/command-line)) before and after the file, but it doesn't seem to work.

```
${plantuml}/bin/plantuml -tsvg -darkmode -filename "arch-dark.uml" arch.uml -o ../_static/arch-dark.svg
```

Since it creates a graphic per file I had to create another file for the dark mode.

Reply:
Ok cool. Please comment that here on this file. Otherwise it's not obvious why it was done this way.

Reply:
I added the info to the [diagram's README](https://github.com/PostgREST/postgrest/blob/f8c7f3b6cee9b8ee4558197fbb17f744ef10e4f7/docs/_diagrams/README.md#uml).